framework/components/ecl-forms/ecl-forms-selects/ecl-forms-selects.twig

```diff
@@ -12,8 +21,8 @@
   {% set _css_class = _css_class ~ ' ecl-select--has-error' %}
 {% endif %}
 
-{% if extra_class is defined %}
-  {% set _css_class = _css_class ~ ' ' ~ extra_class %}
+{% if extra_classes is defined %}
```

Comment:
Sometimes we use `extra_class`, sometimes `extra_classes`. Can we agree to use `extra_classes` everywhere?